queries/muttrc/highlights.scm

```diff
@@ -20,10 +20,11 @@
 
 (regex) @string.regexp
 
-[
-  (option)
-  (command_line_option)
-] @variable.builtin
+(command_line_option) @variable.builtin
+
+((option) @variable.builtin (#not-match? @variable.builtin "^my_"))
```

Comment:
match is unacceptable. either use `#eq?` or `#lua-match!`.

And make sure it's really builtin.

Reply:
There are many queries use `#match?`:

```sh
❯ rg '#match' queries/*/highlights.scm
queries/vala/highlights.scm
33:  (#match? @type "^[A-Z][A-Za-z_0-9]{2,}$"))
115:  (#match? @function.macro "^assert[A-Za-z_0-9]*|error|info|debug|print|warning|warning_once$"))
...
```

Reply:
And most of those are usage where there doesn't exist an equivalent lua pattern

The match pattern in your PR, however, can be translated to lua pattern

Reply:
Fixed.

Reply:
Lint is still failing.